pkg/api/util/db_test.go

```diff
@@ -401,3 +401,38 @@ func TestDeleteVolumeSnapshotDBEntry(t *testing.T) {
 		}
 	})
 }
+
+func TestCreateFileShareSnapshotDBEntry(t *testing.T) {
+	var fileshare = &model.FileShareSpec{
+		BaseModel: &model.BaseModel{
+			Id: "bd5b12a8-a101-11e7-941e-d77981b584d8",
+		},
+		Status: "available",
+	}
+	var req = &model.FileShareSnapshotSpec{
+		BaseModel: &model.BaseModel{
+			Id: "3769855c-a102-11e7-b772-17b880d2f537",
+		},
+		Name:        "sample-snapshot-01",
+		Description: "This is the first sample snapshot for testing",
+		Status:      "available",
+		ShareSize:  int64(1),
+		FileShareId: "bd5b12a8-a101-11e7-941e-d77981b584d8",
+		ProfileId:   "1106b972-66ef-11e7-b172-db03f3689c9c",
+	}
+
+	t.Run("test +ve", func(t *testing.T) {
```

Comment:
Can u add the test case for the name existence?

Reply:
Added name existence test case